### DISCUSSION TITLE
receiving values by mqtt in scripting...

### DISCUSSION THREAD

**Discussion by naksper** [arendst/Tasmota#13068] [Q&A] [answered]
Is there an example to receive values by mqtt in scripting? 

I'm trying to control a slider generated in scripting to be changed by mqtt.

**Comment by barbudor** [comment, marked as answer]
Look for SUPPORT_MQTT_EVENT in the docs for Scripting-language

**Reply by naksper** [reply]
Thank you very much for the quick response, I had already read the documentation and I have also been looking for all the examples on the scripting documentation, but I don't understand how to link a variable with the information coming from mqtt. 
I get success on sending data using mqtt but not receiving...
Surely I am doing something wrong, but not I'm not able to know where is the problem, and that was the reason because I asked for an example.
